static/js/admin/performance_visualizations/helpers.js

```diff
@@ -0,0 +1,584 @@
+            var text = (/(time|throughput|duration)/.test(field)) ? _that.tipType("deposits", d) : _that.tipType("operations", d);
+            _that.tipShow(_that.scatter_tip, text);
+            d3.select(this).attr("r", 9).style("stroke-width", 3);
+        }).on("mouseout", function(d) {
+            _that.tipHide(_that.scatter_tip);
+            d3.select(this).attr("r", 4.5).style("stroke-width", 1);
+        });
+
+    circles.transition().duration(1000)
+        .ease("sin-in-out")
+        .attr("cx", function(d) { return xScale(d.date); })
+        .attr("cy", function(d) { return yScale(d[field]); })
+        .attr("r", 4.5);
+
+    circles.exit().remove();
+
+    return circles;
+};
+
+/**
+ * Create line path function
+ * @param xScale
+ * @param yScale
+ * @param y
+ * @returns {*}
+ */
+CdrGraphs.prototype.lineGenerator = function(xScale, yScale, y) {
+    return d3.svg.line()
+        .interpolate("monotone")
+        .x(function(d) { return xScale(d.date); })
+        .y(function(d) { return yScale(d[y]); });
+}
+
+/**
+ * Add svg path to a chart
+ * @param svg
+ * @param id
+ * @param scale
+ * @param scale
+ * @returns {*}
+ */
+CdrGraphs.prototype.appendPath = function(svg, id, scale, data) {
+    svg.append("path#" + id)
+        .attr("fill", "none")
+        .attr("stroke", "steelblue")
+        .attr("stroke-width", 2.5)
+        .attr("d", scale(data))
+        .translate([this.margins.left, this.margins.top]);
+
+    return svg;
+}
+
+/**
+ * Draw SVG path
+ * @param selector
+ * @param scale
+ * @param data
+ * @returns {*}
+ */
+CdrGraphs.prototype.redrawPath = function(selector, scale, data) {
+    return d3.select(selector).transition()
+        .duration(1000)
+        .ease("sin-in-out")
+        .attr("d", scale(data));
+}
+
+/**
+ * Format large numbers with commas & 2 decimal places or just commas if not a decimal number
+ * @param number
+ * @returns {*}
+ */
+CdrGraphs.prototype.numFormat = function(number) {
+    var format= /\./.test(number) ? d3.format(",.2f") : d3.format(",");
+    return format(number);
+};
+
+/**
+ * Tool tip operations text
+ * @param d
+ * @returns {string}
+ */
+CdrGraphs.prototype.tipTextOperations = function(d) {
+    var text = "<h5 class='text-center smaller'>" + this.stringDate(d.date) + "</h5>";
+
+    text += "<p class='text-center'>Deposit Metrics</p>" +
+        "<ul class='list-unstyled smaller'>" +
+            "<li>" + "Files Ingested: " + this.numFormat(d.throughput_files) + "</li>" +
+            "<li>" + "Total MB Ingested: " + this.numFormat(d.throughput_bytes) + "</li>" +
+            "<li>" + "Avg Filesize (MB): " + this.numFormat(d.avg_filesize) + "</li>" +
+        "</ul>" +
+
+        "<p class='text-center'>Operations Metrics</p>" +
+
+        "<ul class='list-unstyled smaller'>" +
+        "<li>" + "Moves: " + this.numFormat(d.moves) + "</li>" +
+        "<li>" + "Finished Enh: " + this.numFormat(d.finished_enhancements) + "</li>" +
```

Comment:
For this kind of total, we definitely want to make sure it is encompassing all enhancements, not just TechnicalMetadata